pyproject.toml

```diff
@@ -52,6 +52,7 @@ python_version = "3.10"
 show_error_codes = true
 warn_unreachable = true
 enable_error_code = ["ignore-without-code", "redundant-expr", "truthy-bool"]
+disable_error_code = ["attr-defined"]
```

Comment:
I wonder if there is a way for us to type annotate our additions to `pytest` rather than doing this. For instance, what about:

```py
@dataclass
class PytestPyodideExtra:
   pyodide_dist_dir: str
   # ... other fields

def pytest_pyodide_extra() -> PytestPyodideExtra:
   return pytest.pytest_pyodide_extra
```

Also, shouldn't we be adding these extra fields to the pytest config object so they aren't globals? Seems like that would be a better design and make the test mocks simpler. Probably can leave this to a followup though.

Reply:
I think we should not add attributes to pytest and instead manage those variables ourselves, such as:

```py

SINGLETON_CONFIG = {...}

def get_config():
    return SINGLETON_CONFIG

##########

from pytest_pyodide import get_config
config = get_config()
print(config.pyodide_runtimes)
```